### PR TITLE
Added a file poopy

### PR DESCRIPTION
**Pull request by vakodrazan**
I've create a new PR. Please have a look and make some request if need.

Thanks.

### CODE REVIEW

**Review by ndorfin — CHANGES_REQUESTED** [review]
Please fix the spaces around the punctuation

**Comment by vakodrazan** [comment]
I have fixed it.

File: p/poopy.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...ge. She was just 15 years old. Two years later, in 1983, Njila's group of Poopy's friends sought out a female singer, and it was Poopy's father who first met Poopy and Njila, singing Jeanne Moreau's song Poopy At that, Njila was immediately taken captive and the group chose Njila as a group. In 1990, Njila's group moved to France, while Poop chose to stay in Madagascar. That's when he started singing solo (Solo). He worked with Solofo Ranarivelo and in 1990 released his first "Japan koa". Teo audiovisual trips on the water, "Gina Gina" In 1991, a stage which Poopy, Bodo, Thad And Susan, was well known that i "Hidden" Poopy and Bodo, a song composed by Poopy when he was in Njila's band, and" One Power "he sang with Luc and Mighty (Johary) and Bodo. He also started working with Naivo himself, and since then Poopy has created all of his musical tastes and Naivo has set his voice and cares about the art world. 1992Forbidden love, "which confirms the quality of the partnership between Poopy and Naivo. Ary In 1993 he released the album "Victory" and marked nihirany. Teo decade she came contrat with the BMG and went South Africa performed the "Rainbow Rainbow" album, and met Bonnie Tyler.
```
Comment (ndorfin) [inline p/poopy.md]: Please fix errors in the grammar and spelling.

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Comment by vakodrazan** [comment]
Have a look at my PR, I rewrote it.

**Review by ndorfin — APPROVED** [review]
Nice fixes, thanks!

**Pull request merged**